assets/agenda/components/AgendaCoverages.tsx

```diff
@@ -71,6 +73,22 @@ function AgendaCoveragesComponent({item, coverages, wireItems, actions, user, on
                         </span>
 
                     </div>
+                    {<div className='coverage-item__row align-items-center'>
+                        {getCoverageAsigneeName(coverage) && (
```

Comment:
`getCoverageAsigneeName`. `getCoverageAsigneeName` and `getCoverageDeskName` are called multiple times for the 1 render. It would be better to store the results of these functions and use the stored response instead.

Also, if none of those 3 responses have a valid value, then `<div className='coverage-item__row align-items-center'>` should not be rendered

Reply:
yep sounds identical